src/main.js

```diff
@@ -0,0 +1,24 @@
+import UserRepository from './UserRepository'
+
+let userRepo, hydrationRepo, sleepRepo, activityRepo
+
+let fetchData = (dataType) => {
+  return fetch(`http://localhost:3001/api/v1/${dataType}`)
+    .then(res => {
+      return res.ok ? res.json() : console.log(`ERROR with ${dataType} path`)
```

Comment:
There is a interrogation point in this code, not sure about this. Would you mind explaining?

Reply:
This is a Ternary to check if we have a 200 code : if not which fetch path has failed.